gulpfile.js

```diff
@@ -798,6 +802,8 @@ gulp.task('minified-post', gulp.series('minified-pre', function (done) {
                    Terser.minify(pdfFile).code);
   fs.writeFileSync(MINIFIED_DIR + '/build/pdf.worker.min.js',
                    Terser.minify(pdfWorkerFile, optsForHugeFile).code);
+  fs.writeFileSync(MINIFIED_DIR + '/web/pdf.worker.min.js',
```

Comment:
What's the reason for the alternative PDF worker script?

Reply:
Good question.

It just makes it easier to copy over the files into the reader project.  The worker script without this modification ends up in `build/generic/build`, this file is in `build/generic/web`.   Putting it in `build/generic/web` means everything the viewer needs is in `web`, so we can just `rsync` that directory over.